lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -101,6 +102,21 @@
 
                 setExtraData: function(data) {
                     return _.extend(data, this.user_details);
+                },
+
+                validateCountry: function() {
+                    if( !this.model.get('country') ){
+                        var $submissionContainer = $('.submission-error'),
+                            $errorMessageContainer = $submissionContainer.find('.message-copy'),
+                            $countryLabel = $('#user-country-title');
+
+                        $countryLabel.addClass('error');
+                        $errorMessageContainer.append(
+                            "<li>Please go to your <a href='/account/settings'>profile page</a> and provide your country of residence.</li>"
```

Comment:
I think the URL should be passed from Django [here](https://github.com/edx/edx-platform/blob/bderusha/fa-require-country/lms/djangoapps/courseware/views.py#L1540-L1632). This message also needs to be marked with `gettext`.